src/components/QueryEditor.tsx

```diff
@@ -156,6 +159,7 @@ const QueryClause = ({
         <AutocompleteInput
           value={con.target}
           setValue={setConditionTarget}
+          onBlur={(e) => setConditionTarget(e, false)}
```

Comment:
shouldn't this first argument be `con.target` instead of `e` ?

Reply:
the other two seem to be sending an event so those should be fine

Reply:
`setValue` seems to set `con.target` for an autocomplete on hover, even if the user doesn't select it.  So `e` looks to be a more accurate reflection of what the user experiences.

https://github.com/dvargas92495/roamjs-query-builder/assets/3792666/7e818ce3-ddda-4fef-8ef3-42313860765f